### DISCUSSION TITLE
What SDK are most suitable for WordPress?

### DISCUSSION THREAD

**Discussion by jamieblomerus** [percy/cli#1384] [Support]
I found the recommendation to use Percy on [BrowserStack blog](https://www.browserstack.com/guide/wordpress-testing). But I do not find any guide for how to integrate the SDK to our WordPress site.

Could someone please tell me how to integrate Percy with WordPress or at least which SDK to use?

**Comment by rohit-y180** [comment]
Could you please elaborate more on how you intend to carry out the visual testing? Are you using any framework for Automated Testing?
If yes, then you can use the SDK of that framework.

**Reply by jamieblomerus** [reply]
No, I am really new to automated testing in general. Could you recommend any framework?
